op-deployer/pkg/deployer/bootstrap/implementations.go

```diff
@@ -89,6 +93,25 @@ func (c *ImplementationsConfig) Check() error {
 	if c.DisputeGameFinalityDelaySeconds == 0 {
 		return errors.New("dispute game finality delay in seconds must be specified")
 	}
+	// Check V2 fault game parameters only if V2 dispute games feature is enabled
+	// DevFeatureBitmap 0x0100 bit enables V2 dispute game deployment
+	var deployV2FeatureFlag common.Hash
+	deployV2FeatureFlag[30] = 0x01
+	deployV2Games := (c.DevFeatureBitmap[30] & deployV2FeatureFlag[30]) != 0
```

Comment:
This won't be the last time we'll be introducing dev feature flags. So lets introduce a constant for the deployV2Games feature flag as was done in solidity. And you could do a bitwise for a generic flag checker rather than check specific bits:
```go
func isDevFeatureEnabled(bitmap, flag common.Hash) bool {
	b := new(big.Int).SetBytes(bitmap[:])
	f := new(big.Int).SetBytes(flag[:])
	return new(big.Int).And(b, f).BitLen() != 0
}
```